### DISCUSSION TITLE
How to show widget in C?

### DISCUSSION THREAD

**Discussion by bfamzz** [lcallarec/live-chart#31] [Use LiveChart C flavour]
Hello @lcallarec 

Following up on this [thread](https://github.com/lcallarec/live-chart/issues/26#issuecomment-747060919), I have successfully linked the libraries but the widget is not being displayed.

I have tried `gtk_widget_show_all` on the `LiveChart` object and even the main `GtkWindow` but the chart is not visible.

Any suggestions please?

Thanks.

**Comment by bfamzz** [comment]
Solved.

Set the `gtk_widget_set_vexpand` and `gtk_widget_set_hexpand` properties to `TRUE` on the chart.

**Reply by lcallarec** [reply]
Happy to hear that it's finally working !

**Comment by 1ay1** [comment]
I think problem for me is that libgee devel is not available in RHEL, is there some way to bypass the libgee dependency?

**Reply by lcallarec** [reply]
Hi 1ay1, I'm afraid that you can't bypass the libgee dependency.
If devel package in missing in core RHEL package repository, couldn't you use CentOS one ? Or just installing the package from https://centos.pkgs.org/8/centos-powertools-x86_64/libgee-devel-0.20.1-1.el8.x86_64.rpm.html ?
